lib/Runtime/PlatformAgnostic/Platform/Windows/ArraysUtility.cpp

```diff
@@ -0,0 +1,41 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#include "RuntimePlatformAgnosticPch.h"
+#include "Common.h"
+#include "ChakraPlatform.h"
+
+namespace PlatformAgnostic
+{
+namespace Arrays
+{
+
+    size_t Utility::GetLocaleSeparator(WCHAR* szSeparator)
+    {
+
+        LCID lcid = GetUserDefaultLCID();
+        size_t count = 0;
+
+        count = GetLocaleInfoW(lcid, LOCALE_SLIST, szSeparator, 5);
```

Comment:
>5 [](start = 64, length = 1)

Should we pass this constant from the function?

Reply:
Thanks for pointing this out. No need to pass it from the function. The correct way would be replacing it with `sizeof(szSeparator)/sizeof(char16)`.